CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
   unlabelled arguments could not be compiled when targeting JavaScript.
 - Fixed a bug where local variables in case guard constant expressions caused
   the compiler to panic.
+- Insert todo in empty anonymous functions.
```

Comment:
I don't think a reader would know what this means just from reading this. Could we give a little more detail? 🙏

Reply:
@lpil words are my kryptonite 😆 hopefully it makes more sense now.